README.md

```diff
@@ -43,19 +48,9 @@ You can also include it directly from a CDN:
 <script src="https://cdn.jsdelivr.net/npm/@turf/turf@6/turf.min.js"></script>
 ```
 
-You can create light-weight turf builds with only the functions you need using the [turfjs-builder UI](https://turfjs-builder.herokuapp.com/) or using browserify as described below.
-
-### Browserify
```

Comment:
We previously removed some browserify references somewhere else. I don't know that many people use browserify at this point.